coordinator/gscoordinator/coordinator.py

```diff
@@ -48,8 +48,11 @@
 # capture system stdout
 sys.stdout = StdoutWrapper(sys.stdout)
 
+import vineyard
+import vineyard.io
```

Comment:
Put those two `import` into the function itself.

Reply:
done